src/api/spec/services/workflows/yaml_downloader_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
-RSpec.describe Workflows::YAMLDownloader, type: :service do
-  let(:yaml_downloader) { described_class.new(payload) }
+RSpec.describe Workflows::YAMLDownloader, type: :service, vcr: true do
+  let(:yaml_downloader) { described_class.new(payload, token: FactoryBot.build(:workflow_token)) }
```

Comment:
`FactoryBot` shouldn't be needed here.

```suggestion
  let(:yaml_downloader) { described_class.new(payload, token: build(:workflow_token)) }
```

Reply:
Done.